pkg/webui/console/containers/side-bar/navigation/app-side-navigation.js

```diff
@@ -67,30 +71,30 @@ const AppSideNavigation = () => {
           <DedicatedEntity
             label={entityId}
             buttonMessage={m.buttonMessage}
-            icon="arrow_back"
             className="mt-cs-xs mb-cs-l"
-            path={`/applications`}
+            path={`/applications/${appId}`}
```

Comment:
This had to be refactored throughout because for some reason after the refactor, the link would be rendered as `applications/applications/app-id`.
Must've something to do with how react router determines the current route by calculating all `<Route />`'s up the render hierarchy.